frontend/src/content/news.json

```diff
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-06-30",
+      "title": "Metabolic Atlas 3.0",
+      "text": "<b>Metabolic Atlas 3.0</b> is out! In this major release we provide access to the newly created database <i>GotEnzymes</i>. This contains over 25,794,195 million predicted k<sub>cat</sub> from 8,099 species. The values are predicted by cutting-edge artificial intelligence tools. For more information the prediction methods, please have a look at <a href='https://doi.org/10.1038/s41929-022-00798-z'><i>Deep learning-based k<sub>cat</sub> prediction enables improved enzyme-constrained model reconstruction</i></a>. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.0' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

Comment:
I would suggest 
1. `by cutting-edge artificial intelligence tools` => `by the cutting-edge artificial intelligence tools`
2. `For more information the prediction methods` => `For more information of the prediction methods`

Reply:
I don't think the extra `the` is an improvement for (1).
For (20), my suggestion is slightly different: `For more information *about* the prediction methods`.

Reply:
An "about" is added in 9e3032a.
Since there seems to be a majority against the extra "the", I did not include it.